content/en/agent/kubernetes/tag.md

```diff
@@ -350,6 +350,50 @@ container_env_as_tags:
 {{% /tab %}}
 {{< /tabs >}}
 
+## Container labels as tags
+
+Starting with Agent v7.33+, the Agent can collect container labels and use them as tags to attach to all the metrics corresponding to the container.
+Both `docker` and `containerd` containers are supported. In the case of `containerd` the minimum supported version is v1.5.6 because previous releases didn't propagate labels correctly.
```

Comment:
```suggestion
The Agent can generate tags from container labels for both `docker` and `containerd` containers. In the case of `containerd`, the minimum supported version is v1.5.6, because previous releases do not propagate labels correctly.
```